packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -1384,4 +1339,19 @@ export default class ScheduledPaymentModule {
     const scheduledPaymentConfig = new Contract(configModuleAddress, ScheduledPaymentConfigABI, this.ethersProvider);
     return await scheduledPaymentConfig.validForDays();
   }
+
+  private composeFees(
+    feeFixedUSD: number,
+    feePercentage: number
+  ): { fixedUSD: { value: string }; percentage: { value: string } } {
```

Comment:
nit: typescript is smart enough to infer the return of this function, but if we do want to enforce it, I'd suggest to extract this type to improve readability.

Reply:
I choose to remove it. Thanks